### DISCUSSION TITLE
ressource en reflet qui n'apparait pas

### DISCUSSION THREAD

**Discussion by MickaGaffet** [witsa/synapps#760] [Q&A] [answered]
Bonjour, 

J'ai un soucis sur un Synapp pour une visu de SST sur téléphone en gros je lis une liste de reflet sur un ensemble,
![image](https://github.com/user-attachments/assets/9f67b17c-4137-43bd-aaf6-92be766634dc)

La lecture des ressource sur un plug 911 et 903 pour une AI fonctionne dans l'exécution dans le logiciel/le navigateur  mais quand je suis en direct sur l'automate sa ne s'affiche plus 
![image](https://github.com/user-attachments/assets/2a1e399a-b8f5-4340-a3b4-0b3ae6dd155f)

le synapps a bien étais publier sur la capture d'écran au dessus c'est le même synapps la seul différence c'est que le 1er est en Localhost par l'appli et le 2de je suis sur l'automate en IP

Merci

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour,

La différence peut venir des utilisateurs utilisés et des groupes auxquels ils appartiennent.

**Reply by MickaGaffet** [reply]
Effectivement c'étais ce qui me bloquée 

Merci, Bonne journée
